File: src/pooh3/mobi/app/a01_clock/model/DegreeCalcService.java
package pooh3.mobi.app.a01_clock.model;

public class DegreeCalcService {

    public Degree degreeBetweenTwoHands(Clock c) {
        return c.longHand.degree()
                .minusMin(c.shortHand.degree(c.longHand));
    }
}

File: src/pooh3/mobi/app/a01_clock/model/DegreeCalcService.kt
package pooh3.mobi.app.a01_clock.model

class DegreeCalcService {

    fun degreeBetweenClockTwoHands(c: Clock): Degree {
        return c.longHand.degree()
                .minusMin(c.shortHand.degree(c.longHand))
    }
}
